scipy/stats/tests/test_stats.py

```diff
@@ -1771,6 +1771,30 @@ def test_zscore_ddof(self):
         assert_array_almost_equal(z[0], z0_expected)
         assert_array_almost_equal(z[1], z1_expected)
 
+    def test_zscore_nan_propagate(self):
+        # Test nan_policy set to 'propogate'
+        x = np.array([1, 2, np.nan, 4, 5])
+
+        z = stats.zscore(x, nan_policy='propagate')
+
+        expected = np.array([np.nan, np.nan, np.nan, np.nan, np.nan])
+        assert_array_equal(z, expected)
+
+    def test_zscore_nan_omit(self):
+        # Test nan_policy set to 'omit'
+        x = np.array([1, 2, np.nan, 4, 5])
+
+        z = stats.zscore(x, nan_policy='omit')
+
+        expected = np.array([-1.2649110640673518, -0.6324555320336759, np.nan, 0.6324555320336759, 1.2649110640673518])
```

Comment:
This line is 119 characters long; would be good make it shorter by splitting it up:

```python
expected = np.array([
    -1.2649110640673518,
    -0.6324555320336759,
    np.nan,
    0.6324555320336759,
    1.2649110640673518
])
```

Reply:
Thank you.